modules/apps/document-library/document-library-web/src/main/java/com/liferay/document/library/web/internal/display/context/DLAdminManagementToolbarDisplayContext.java

```diff
@@ -1228,6 +1113,73 @@ private boolean _isStagedActions() {
 		return false;
 	}
 
+	private void _setFilterParameters(PortletURL portletURL) {
+		long[] assetCategoryIds = ArrayUtil.toLongArray(
+			_getSelectedAssetCategoryIds());
+
+		if (ArrayUtil.isNotEmpty(assetCategoryIds)) {
+			portletURL.setParameter(
+				"assetCategoryId", ArrayUtil.toStringArray(assetCategoryIds));
+		}
+		else {
+			portletURL.setParameter("assetCategoryId", (String)null);
+		}
```

Comment:
Why do you need to check for emptiness here? If the array is empty, ArrayUtil.toStringArray will return the empty string (not null)? Do you need this to be null for some specific reason? The same applies to all other usages here.

Reply:
I'm not sure if `null` value is really needed. I just replicated how it was working before. I need to test if `null` is needed or not.

Reply:
After testing I simplified the method by removing all emptiness checks. Done in new commit https://github.com/liferay-lima/liferay-portal/pull/4378/commits/61f9b529e3854f535b115fd42eb3f102e51fd7a8